bin/ch-run.c

```diff
@@ -303,59 +149,16 @@ bool get_first_env(char ** array, char ** name, char ** value)
    return false;
 }
 
```

Comment:
Looks like these lines need to be moved to charliecloud.c? Aren't they part of `enter_udss()`?

Reply:
No, it's to gather environment variables (see above).